forte/data/data_store.py

```diff
@@ -53,26 +60,15 @@ def __init__(self):
         The values are the index of attributes among sortedlist.
```

Comment:
Check the grammar of the docstring. "The values are the indices". Please check grammar for the rest of the docstring.